proof/refine/AARCH64/VSpace_R.thy

```diff
@@ -615,15 +562,13 @@ lemma vcpuSave_corres:
                       apply (rule corres_Id; wpsimp)
                      apply (wpsimp wp: mapM_x_wp_inv hoare_vcg_imp_lift'
                                  simp: if_apply_def2)+
-  done *)
+  done
 
 lemma vcpuDisable_corres:
-  "corres dc (\<lambda>s. (\<exists>v. vcpuopt = Some v) \<longrightarrow> vcpu_at  (the vcpuopt) s)
+  "corres dc (\<lambda>s. (\<exists>v. vcpuopt = Some v) \<longrightarrow> vcpu_at  (the vcpuopt) s \<and> True)
```

Comment:
Need to check, it might be an artefact of some lifting that is applied later (to provide something to match a rule against). If that is the case, it should at least get a comment, though.